src/ripple/test/JSONRPCClient.h

```diff
@@ -29,7 +29,7 @@ namespace test {
 
 /** Returns a client using JSON-RPC over HTTP/S. */
 std::unique_ptr<AbstractClient>
-makeJSONRPCClient(Config const& cfg);
+makeJSONRPCClient(Config const& cfg, unsigned rpc_version = 2);
```

Comment:
Would `@param`s docs be worthwhile here? It's pretty obvious by the variable names, but I don't know how thorough we want to be.